### ISSUE TITLE
Valuedump - Not selling

### ISSUE THREAD

**Issue by Agorash** [ISeeDEDPpl/Questor#281]
Even when price shows at 0.0% or better price it resolves two type of results depending if window is on simple or advanced mode.
Simple: ITEM [ X units ] is worth more as mins [Refine ...][Sell each..]etc.
Advanced: No order available for 'ITEM'
Sometimes it sells but I checked about 400 or 500 items and more than 15% can be sold by 0.0% regional average or above. I don't know why happens. I updated two times InvTypes but still doing this.

**Comment by ISeeDEDPpl** [comment]
is the
`
Simple: ITEM [ X units ] is worth more as mins [Refine ...][Sell each..]etc.
`
output wrong? It should be something like:

`
[Hornet I[1units] is worth more as mins [Refine each: 5,630.10][Sell each: 330.03][Refine total: 5,630.10][Sell total: 330.03]
`
I need to know if the math is somehow wrong or if the math is correct but the decision it made based on the math is wrong... 

the sell window should be in simple mode and will not work correctly unless it is in that mode, we cannot put it in that mode programatically so it must be left in that mode before valudeump starts. 

I'll look into this in more detail over the weekend, but clarification on where it is doing the wrong thing would be appreciated.

**Comment by Agorash** [comment]
Sorry, I edited too much that line. It looks like yours.
I tried many times with many variations with the checkbox, also underselling and not refining but stills not selling.
First time I started yesterday It started to sell like a machinegun, not selling devaluated items of course, but many time later stopped and not worked again. Maybe need very big balance above average price? My "green" prices above don't goes more than... 1% or 2%. Mostly 0.0% or 0.x%. Anyway Im very intrigued why when I check undersell it don't sell nothing :s 

I take this post to ask you about one thing related. Will valuedump include in the future a textbox or combobox to limit undersell? I mean Im ok selling any item with -5% but obviously not with -200% or rares -5000%. I don't know if undersell has restrictions but it would be great to define a limite by user :)

Thanks for your attention
